src/api/app/models/user.rb

```diff
@@ -129,6 +130,22 @@ def create_home_project
     self.login_failure_count = 0 if login_failure_count.nil?
   end
 
+  def self.autocomplete_login(prefix = '')
```

Comment:
For this you can just use an `alias`

Reply:
why is actually this method needed? why not just using `autocomplete_backend` directly? as @mdeniz it is just the same thing with different name.... :confounded: